# -*- coding:utf-8 -*-
# $ catkin_make -DCATKIN_WHITELIST_PACKAGES="copley"

# Control
float32 motor_l
float32 motor_r